extension/prefs.js

```diff
@@ -161,9 +161,15 @@ class HamsterSettingsWidget extends Gtk.Grid {
     }
 
     _onHotkeyChange(widget, bananas) {
-        //global.log(widget, bananas);
         let hotkey = widget.get_text();
-        let [key, mods] = Gtk.accelerator_parse(hotkey);
+        let [key, mods] = [null, null];
+
+        if (Gtk.MAJOR_VERSION == 4) {
```

Comment:
I'd say it might be better to assume that it won't revert to old behavior with versions higher than 4:

```suggestion
        if (Gtk.MAJOR_VERSION >= 4) {
```